### PR TITLE
Add core weekly 25 of 2020

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

File: news/_posts/core-weekly/2020-06-22-coreweekly-25-2020.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 15th to Sunday 21th of June 2020.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+PrestaShop eCommerce Week is one week away! Will you join us ? This is a 5-day live session program!
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-06-22-coreweekly-25-2020.md]:
```suggestion
PrestaShop eCommerce Week is one week away! Will you join us? This is a 5-day live session program!
```

File: news/_posts/core-weekly/2020-06-22-coreweekly-25-2020.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+* [#19840](https://github.com/PrestaShop/PrestaShop/pull/19840): Functional tests - Add id to forms buttons in design page and category, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#19836](https://github.com/PrestaShop/PrestaShop/pull/19836): Functional tests - Improve selectors for email theme page, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#19835](https://github.com/PrestaShop/PrestaShop/pull/19835): Functional tests - Add ids to forms in payment preferences page, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#19823](https://github.com/PrestaShop/PrestaShop/pull/19823): Functional tests - Improve grid header selectors (search and reset), by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#19805](https://github.com/PrestaShop/PrestaShop/pull/19805): Functional tests - Add forms ids and fix them for tests, by [@boubkerbribri](https://github.com/boubkerbribri)
+
+
+### Merge
+* [#19804](https://github.com/PrestaShop/PrestaShop/pull/19804): Merge 177 to develop - 16/06/20, by [@jolelievre](https://github.com/jolelievre)
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-06-22-coreweekly-25-2020.md]:
```suggestion
* [#19804](https://github.com/PrestaShop/PrestaShop/pull/19804): Merge 177 to develop - 16/06/2020, by [@jolelievre](https://github.com/jolelievre)
```

File: news/_posts/core-weekly/2020-06-22-coreweekly-25-2020.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 25 of 2020"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2020-06-22
```
Comment (matks) [inline news/_posts/core-weekly/2020-06-22-coreweekly-25-2020.md]:
```suggestion
date:   2020-06-23
```

**Review by Progi1984 — APPROVED** [review]

File: news/_posts/core-weekly/2020-06-22-coreweekly-25-2020.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+## General messages
+
+Dear developers,
+
+PrestaShop eCommerce Week is one week away! Will you join us? This is a 5-day live session program!
+Sixty live sessions by experts will cover multiple topics, be it PrestaShop employees or from the PrestaShop community.
+
+You can discover more [if you follow this link](https://events.prestashop.com/prestashop-ecommerce-week).
+
+In the meantime, [feedbacks continue to arrive on GitHub](https://github.com/PrestaShop/PrestaShop/projects/7) about the 1.7.7 beta and more and more bug fixes are fixed, which means [Release Candidate 1](https://github.com/PrestaShop/PrestaShop/issues/18647) is closer and close to be delivered...
```
Comment (matks) [inline news/_posts/core-weekly/2020-06-22-coreweekly-25-2020.md]:
```suggestion
In the meantime, [feedbacks continue to arrive on GitHub](https://github.com/PrestaShop/PrestaShop/projects/7) about the 1.7.7 beta and more and more bug fixes are merged, which means [Release Candidate 1](https://github.com/PrestaShop/PrestaShop/issues/18647) gets closer and closer to the release date...
```

**Comment by Progi1984** [comment]
Thanks @matks

**Pull request merged**